### PR TITLE
ENG-1399: Redesigns integration details page to explicitly show DB tables

### PR DESCRIPTION
**Pull request by vsreekanti**
## Describe your changes and why you are making these changes

Previously, we had a slightly opaque way of showing previews of the data tables on the integration details page behind a select menu. It was not clear to users what that menu was for or what would happen if you selected something. 

This PR slightly redesigns that interface to instead show an accordion, in which clicking on each item shows you a preview of the data. You can see a side-by-side comparison in the demo video below.

[Demo video](https://www.loom.com/share/b914d0f0507a457fbabb657b4ef13ccf)

## Related issue number (if any)

ENG-1399

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [N/A] If this is a new feature, I have added unit tests and integration tests.
- [N/A] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

File: src/ui/common/src/components/integrations/integrationObjectList.tsx [hunk 1]
```diff
@@ -74,75 +77,87 @@ const IntegrationObjectList: React.FC<Props> = ({ user, integration }) => {
...
+      : listObjectNamesState.names.length;
+  for (let i = 0; i < tableListLimit; i++) {
+    const element = (
+      <Accordion
+        expanded={openPanel === i}
+        sx={{ width: '100%' }}
+        key={i}
+        onChange={() => handleChange(i)}
+      >
+        <AccordionSummary sx={{ backgroundColor: theme.palette.gray[50] }}>
```
Comment (agiron123) [inline src/ui/common/src/components/integrations/integrationObjectList.tsx:105]: Could we maybe make the accordion a lighter color? maybe gray 25?

**Review by agiron123 — APPROVED** [review]
Neat demo!

I take it only one table can be opened at a time (that's at least what I gleaned from reading the code) correct?

:shipit:

**Comment by vsreekanti** [comment]
Yep, done! @agiron123, let me know what you think:
image

Also added a link to our documentation which I've been meaning to do for a while.

**Comment by vsreekanti** [comment]
@agiron123, yeah, I've actually disabled opening multiple tables at once right now. We have a couple components we use under the hood for this stuff, so the state is managed through redux, which makes it impossible to have multiple tables open at once without significantly refactoring the state management, which I didn't want to dive into here.

**Comment by agiron123** [comment]
> @agiron123, yeah, I've actually disabled opening multiple tables at once right now. We have a couple components we use under the hood for this stuff, so the state is managed through redux, which makes it impossible to have multiple tables open at once without significantly refactoring the state management, which I didn't want to dive into here.

Sounds good. I was hoping that only one table could be open at a time.

**Pull request merged**